sdk/opendp/whitenoise/synthesizers/mwem/mwem.py

```diff
@@ -23,15 +22,17 @@ class MWEMSynthesizer(BaseSynthesizer):
     Linear queries used for sampling in this implementation are
     random contiguous slices of the n-dimensional numpy array. 
     """
-    def __init__(self, Q_count=400, epsilon=3.0, iterations=30, mult_weights_iterations=20):
+    def __init__(self, Q_count=400, epsilon=3.0, iterations=30, mult_weights_iterations=20, splits = [], split_factor=-1):
         self.Q_count = Q_count
         self.epsilon = epsilon
         self.iterations = iterations
         self.mult_weights_iterations = mult_weights_iterations
         self.synthetic_data = None
         self.data_bins = None
         self.real_data = None
-
+        self.splits = splits
+        self.split_factor = split_factor
+        
     def fit(self, data, categorical_columns=tuple(), ordinal_columns=tuple()):

```

Comment:
Should the two optional params  default to None or tuple?

Reply:
good catch, meant to remove these